doc_site/extensions/man.js

```diff
@@ -0,0 +1,28 @@
+/* Example use
+ *
+ * man:page[N,external]
+ *
+ *  N: section
+ *  external: optional arg, links to external man page
+ *
+ */
+module.exports = function(registry) {
+    registry.inlineMacro('man', function() {
+        var self = this
+        self.process(function (parent, target, attrs) {
+            let section = attrs.$positional[0] === undefined ? '' : attrs.$positional[0];
+            let internal = attrs.$positional[1] === undefined ? true : false
+            const attributes = {}
+            const content = `${target}(${section})`
+            let url
+            if (internal) {
+                url = `${target}.${section}.html`
+            } else {
+                url = `https://manpages.ubuntu.com/man${section}/${target}.${section}.html`
```

Comment:
It's not quite so easy because it then needs to not be an anchor/link object but a totally separate type in the path (probably like an inline bold or something); that's why it's easier to just point it to a URL :)  I also considered pointing to a search engine, but I suppose you have the same problem of which search engine.

Is the issue this is a perceived endorsement of a particular distribution?  I can figure it out, but if there's an agreeable URL that would be easier.

Reply:
Dracut manpages link to each other, but for external man page references marked as “external” in the macro (like teamd.conf(5)) they link to this url.  The Ubuntu collation just seemed complete and easy to link to.  It was just a nice to have and made things a bit simpler and easier